lib/fs.js

```diff
@@ -319,39 +329,46 @@ fs.existsSync = function(path) {
   }
 };
 
+const defaultReadFileOpts = Object.create(null, {
+  flag: { value: O_RDONLY, writable: true }
+});
```

Comment:
I was talking about property access, not object creation. Shallow-cloning an object created with `Object.create(StorageObject.prototype, ...)` (so it has fast properties) is much faster, since slow properties are only faster for map-like usage:

<details><summary>benchmark</summary>

```js
const { shallowClone } = process.binding('util');
const { constants: { O_RDONLY } } = require('fs');

const n = 1e7;

function StorageObject() {}
StorageObject.prototype = Object.create(null);

const direct = Object.create(null, {
  flag: { value: O_RDONLY, writable: true }
});
const proxied = Object.create(StorageObject.prototype, {
  flag: { value: O_RDONLY, writable: true }
});

console.log(%HasFastProperties(direct)); // false
console.log(%HasFastProperties(proxied)); // true

function cloneDirect() { return shallowClone(direct); }
function cloneProxied() { return shallowClone(proxied); }

// warmup
for (var i = 0; i < n; i++)
  cloneDirect();
for (var i = 0; i < n; i++)
  cloneProxied();

console.time('direct');
for (var i = 0; i < n; i++)
  cloneDirect();
console.timeEnd('direct');

console.time('proxied');
for (var i = 0; i < n; i++)
  cloneProxied();
console.timeEnd('proxied');
```

```
false
true
direct: 624.346ms
proxied: 369.772ms
```
</details>